### PR TITLE
Feat: Implement insert images into annotation

### PR DESCRIPTION
**Pull request by shun2wang**
Replace https://github.com/jasp-stats/jasp-desktop/pull/4252

Fix https://github.com/jasp-stats/INTERNAL-jasp/issues/734

### CODE REVIEW

File: Desktop/html/index-jasp.html [hunk 1]
```diff
@@ -25,6 +25,7 @@
 	<script type="text/javascript" src="js/mathjax-tex-svg-full.js">   	  </script>
 	<script type="text/javascript" src="js/highlight.min.js">           </script>
 	<script type="text/javascript" src="js/highlight-r.min.js">        </script>
+	<script type="text/javascript" src="js/quill-blot-formtter.min.js">  </script>
```
Comment (akashrajkn) [inline Desktop/html/index-jasp.html]: Typo here. Should be `quill-blot-formatter.min.js`

**Review by akashrajkn — APPROVED** [review]
Looks good. 
For reference, can you add a comment from where you got the minified js file.

I haven't made a build, so I didn't test this functionally. Can someone else test this functionally?

**Comment by EJWagenmakers** [comment]
This would be great ti have, awesome!

**Comment by shun2wang** [comment]
>For reference, can you add a comment from where you got the minified js file.

Actually I had included reference in the header of `min.js` file of the GitHub repo ( see [readme](https://github.com/Fandom-OSS/quill-blot-formatter#script-tag)) that I got the `min.js` file exported from the `npm` packages. also we can reference a js cdn but we are not a web application so I didn't do that, then let's do so:-)

**Comment by JorisGoosen** [comment]
Works real nice, but in dark mode it displays the image colors inverted.
This can be either a bug or a feature...

it is done to make the plots looks better, but I guess the quill annotation images should be exempt

**Comment by JorisGoosen** [comment]
image

**Comment by shun2wang** [comment]
Ah! I forgot about dark mode again, haha, this is indeed a unexpected behavior.

**Comment by shun2wang** [comment]
image
image

**Comment by JorisGoosen** [comment]
Nice!

Just running a macos build to double check and i can probably merge it after that.

**Pull request merged**